tables/es-chardefs.cti

```diff
@@ -186,6 +185,7 @@ lowercase   \x03C9        4-2456              ω                   GREEK SMALL L
 # Unicode 2000..206F  General Punctuation
 # ----------------------------------------------------------------------------------------------
 
+punctuation \x2012        25-25               ‒                   FIGURE DASH
```

Comment:
@ultraleetj Anything to say about this change?

Reply:
> 
> 
> @ultraleetj Anything to say about this change?

Figure dash is the same as in U.s. English according to the source I had referenced earlier.